contracts/ConsortiumSet.sol

```diff
@@ -239,38 +269,49 @@ contract ConsortiumSet is SystemValidatorSet, InitialConsortiumSet {
 
     // PRIVATE UTILITY FUNCTIONS
     // @notice Add a status tracker for unknown validator.
-    function newStatus(address validator) private hasNoVotes(validator) {
-        validatorsStatus[validator] = ValidatorStatus({
-            isValidator: false,
-            index: pendingList.length,
-            support: AddressVotes.Data({ count: 0 }),
-            supported: new address[](0),
-            benignMisbehaviour: AddressVotes.Data({ count: 0 })
-        });
+    function newStatus(address validator) private returns (ValidatorStatus storage) {
+        // If validator has no votes, create a record for it
+        if ( AddressVotes.count(validatorsStatus[validator].support) == 0 ) {
+            validatorsStatus[validator] = ValidatorStatus({
+                isValidator: false,
+                index: pendingList.length,
+                support: AddressVotes.Data({ count: 0 }),
+                supported: new address[](0),
+                benignMisbehaviour: AddressVotes.Data({ count: 0 })
+            });
+        }
+
+        return validatorsStatus[validator];
```

Comment:
You ended up getting this working?

Reply:
ya the problem was the modifier was using an if-conditional execution instead of require. Wasn't catching the error